modules/user-access/pages/jwt-token.adoc

```diff
@@ -492,3 +492,63 @@ gadmin restart -y
 ====
 Existing environment variable may have some influence on the current use of TigerGraph.
 ====
+
+== Usage of GSQL JWT Token
+Starting from 4.1.0, the token generated by GSQL API is JWT Token and can be used on GSQL and RESTPP server for authentication.
+
+=== create JWT Token
+
+[console]
+----
+POST :14240/gsql/v1/tokens payload '{"secret":"<secret>", "graph":"<graph name>, lifetime: <number in ms>"}'
+----
+
+ * `secret`: the `secret` represents the user who owns so there is no need user login if secret set in the payload
+    if there is no secret specified in the payload a secret will be generated automatically
+                  
+ * `graph`: the `graph` means the scope the token can be used
+    if there is no graph specified the token is global token which is applied to all endpoints
+    
+ * `lifetime`: the `lifetime` means the duration of token can be used and  the time unit is in ms 
+    if there is no lifetime specified then the default time is 7 days
+
+==== Required Privilege:
+If `secret` is specified in the payload, there is no need for user login and if `graph` is specified in the payload, the user must have access to the data of the graph.
+
+=== drop JWT Token
+Since the JWT token cannot be deleted as it has been created, there is a block list to include JWT tokens that are restricted to access on RESTPP and GSQL server.
+
+Clear drop list: clear current user’s block list.
+
+[console]
+----
+DELETE :14240/gsql/v1/tokens?clear=true" 
+----
+
+Drop specific JWT Tokens: add JWT tokens to current user’s block list.
+
+[console]
+----
+DELETE :14240/gsql/v1/tokens"  -d '{"tokens":"$token1, $token2, …"}' 
+----
+
+==== Required Privilege:
+There is no privilege needed since drop JWT token operation is for current user who logins.
```

Comment:
For more clarity: No privilege is required to drop JWT tokens, as this operation is limited to the current logged-in user.

Reply:
@arun-TG-PM  We have addressed your comments, please help to review it again, thanks!